eng/scripts/Create-Apiview-Token-Cadl.ps1

```diff
@@ -78,7 +79,13 @@ if ($revs)
         # initialize git clone if current review is generated from different repo than previous one
         if ($GitRepoName -ne $prevRepo)
         {
-            git clone --no-checkout --filter=tree:0 "https://github.com/$GitRepoName"
+            $gitUrl = "https://github.com"
+            if ($GitPat)
+            {
+                $gitUrl = "https://$GitPat@github.com"
+            }
+
+            git clone --no-checkout --filter=tree:0 "$gitUrl/$GitRepoName"

```

Comment:
Are we doing clones on top of each other if there is multiple different repos? i.e. one from private and one from public? We might need to make sure we clone into unique directories.

Reply:
Looks like this is not intended for this PR but instead for https://github.com/Azure/azure-sdk-tools/pull/4979.